lib/rest/Twilio.js

```diff
@@ -127,6 +127,9 @@ function Twilio(username, password, opts) {
   this.password = password || env.TWILIO_AUTH_TOKEN;
   this.accountSid = opts.accountSid || this.username;
   this._httpClient = opts.httpClient;
+  if (opts.lazyLoading == undefined) {
+    opts.lazyLoading = true;
+  }
   if (!opts.lazyLoading) {
```

Comment:
nit: Could simplify to just:
```node
if (opts.lazyLoading === false) {
   this._httpClient = this.httpClient;
}
```

Reply:
I assume the same change will apply to Line 187 too?

Reply:
Yes.